src/state-summary/state-card-content.html

```diff
@@ -13,40 +13,40 @@
 <link rel="import" href="state-card-weblink.html">
 
 <script>
-Polymer({
-  is: 'state-card-content',
-
-  properties: {
-    hass: {
-      type: Object,
-    },
-
-    inDialog: {
-      type: Boolean,
-      value: false,
+  Polymer({
```

Comment:
All this indentation messes up the diff, making it difficult to see what has changed.